sample/HelloWorld/static/js/helloworld.js

```diff
@@ -6,22 +6,10 @@ var publisher = OT.initPublisher('publisher');
 
 // Attach event handlers
 session.on({
-
-  // This function runs when session.connect() asynchronously completes
-  sessionConnected: function(event) {
-    // Publish the publisher we initialzed earlier (this will trigger 'streamCreated' on other
-    // clients)
-    session.publish(publisher, function(error) {
-      if (error) {
-        console.error('Failed to publish', error);
-      }
-    });
-  },
-
-  // This function runs when another client publishes a stream (eg. session.publish())
+  // This function runs when another client publishes a stream (by calling Session.publish())
```

Comment:
Would you be able to remove the changes to the sample JS from this PR? We currently have a plan to unify the JS samples in all the different SDKs. I'd prefer to keep this PR focused on the documentation changes. Thanks